src/components/settings/SettingsAccessTab/SettingsAccessLinkManagement.jsx

```diff
@@ -42,6 +44,12 @@ const SettingsAccessLinkManagement = ({
   const [isLinkManagementAlertModalOpen, setIsLinkManagementAlertModalOpen] = useState(false);
   const [isLoadingLinkManagementEnabledChange, setIsLoadingLinkManagementEnabledChange] = useState(false);
   const [hasLinkManagementEnabledChangeError, setHasLinkManagementEnabledChangeError] = useState(false);
+  const formattedLinkExpirationDate = useMemo(() => moment.max(
+    [
+      ...couponsData.results.map(coupon => moment(coupon.endDate)),
```

Comment:
Nit: Could you leave a comment to our future selves with a bit more about the why we calculate it this way?